### PR TITLE
Remove key overhaul and some unrelated wording

### PR DESCRIPTION
**Pull request by Bettelstab**
Applies UI changes to remove-key request. Also updates some wording in derive-address and change-passphrase.

### CODE REVIEW

File: src/request/remove-key/index.html [hunk 1]
```diff
@@ -111,4 +140,5 @@ <h1 data-i18n="export-file-heading" class="nq-h1">Download Key File</h1>
 
     </div>
 </body>
-</html>
+
+</html>
```
Comment (sisou) [inline src/request/remove-key/index.html]: I find it fascinating how your editor just can't seem to keep a trailing line-break... :question: :confused:

File: src/translations/index.js [hunk 2]
```diff
@@ -230,15 +230,14 @@ const TRANSLATIONS = {
...
-                               + 'oder zumindest deine Wiederherstellungswörter für dich zugänglich sind.',
-        'remove-key-intro-text-red': 'Falls du zu beiden keinen Zugang hast besteht keine Möglichkeit wieder '
-                                   + 'Zugriff auf dein Wallet zu erhalten.',
-        'remove-key-key-file': 'Schlüsseldatei',
-        'remove-key-download-key-file': 'Schlüsseldatei herunterladen',
-        'remove-key-show-recovery-words': 'Wiederherstellungswörter anzeigen',
+        'remove-key-heading': 'Verliere deinen Zugang nicht',
+        'remove-key-intro-text': 'Falls du dich ausloggst, ohne dein Konto zu sichern, wirst du unwiderruflich'
+                                + 'den Zugriff darauf verlieren. ',
+        'remove-key-login-file': 'Schlüsseldatei',
```
Comment (sisou) [inline src/translations/index.js]:
```suggestion
        'remove-key-login-file': 'LoginDatei',
```

File: src/translations/index.js [hunk 3]
```diff
@@ -102,7 +102,7 @@ const TRANSLATIONS = {
 
         'remove-key-heading': 'Don\'t lose access',
         'remove-key-intro-text': 'If you log out without saving your account, you will irretrievably'
-                                + 'lose access to it!',
+                                + ' lose access to it!',
```
Comment (sisou) [inline src/translations/index.js]: We usually put the space at the end of the first line instead.

File: src/request/remove-key/index.html [hunk 4]
```diff
@@ -72,28 +74,99 @@
...
+                        <defs>
+                        <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(68.6666 116.277) rotate(-180) scale(67.3333 114.941)">
+                        <stop stop-color="#FD6216"/>
+                        <stop offset="1" stop-color="#FC8701"/>
+                        </radialGradient>
+                        <clipPath id="clip0">
+                        <rect width="44.125" height="5.33333" fill="white" transform="translate(12.8889 9.55469)"/>
+                        </clipPath>
+                        </defs>
+                        </svg>
```
Comment (sisou) [inline src/request/remove-key/index.html]: Here is a minified version of this SVG:
```svg
<svg width="70" height="118" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="70" height="117.6" rx="3.4" fill="#fff"/><path d="M1.3 3.6c0-1.2 1-2.3 2.3-2.3h62.8c1.2 0 2.3 1 2.3 2.3V114c0 1.3-1 2.3-2.3 2.3H3.6c-1.2 0-2.3-1-2.3-2.3V3.6z" fill="url(#paint0_radial)"/><g opacity=".5" fill="#fff"><g clip-path="url(#clip0)"><path d="M18.8 12l-1.2-2.2c-.1-.2-.3-.2-.5-.2h-2.5c-.1 0-.3 0-.4.2L13 12v.5l1.2 2.1c.1.2.3.3.4.3h2.5c.2 0 .4-.1.5-.3l1.2-2.1V12z"/><rect x="20.5" y="10.3" width="14.9" height="3.7" rx="1.9"/><rect x="37.3" y="10.3" width="18.7" height="3.7" rx="1.9"/></g><rect x="26.1" y="19.4" width="17.7" height="1.9" rx=".9"/></g><g opacity=".5" fill="#fff"><path fill-rule="evenodd" clip-rule="evenodd" d="M28.2 81.6H20c-.7 0-1.3-.6-1.3-1.3V72c0-.8.6-1.4 1.3-1.4h8.2c.7 0 1.4.6 1.4 1.4v8.2c0 .7-.7 1.3-1.4 1.3zm-6.5-8.1c-.2 0-.3.1-.3.3v4.8c0 .1.1.3.3.3h4.8c.2 0 .3-.2.3-.3v-4.8c0-.2-.1-.3-.3-.3h-4.8zM20 92.5h8.2c.7 0 1.4.6 1.4 1.4v8.1c0 .8-.7 1.4-1.4 1.4H20c-.7 0-1.3-.6-1.3-1.4V94c0-.8.6-1.4 1.3-1.4zm6.5 8.2c.2 0 .3-.2.3-.4v-4.7c0-.2-.1-.4-.3-.4h-4.8c-.2 0-.3.2-.3.4v4.7c0 .2.1.4.3.4h4.8zM41.8 70.7H50c.7 0 1.3.6 1.3 1.4v8.2c0 .7-.6 1.3-1.3 1.3h-8.2c-.7 0-1.4-.6-1.4-1.3V72c0-.8.7-1.4 1.4-1.4zm6.5 8.2c.2 0 .3-.2.3-.3v-4.8c0-.2-.1-.3-.3-.3h-4.8c-.2 0-.3.1-.3.3v4.8c0 .1.1.3.3.3h4.8z"/><path d="M32.3 76.5h1.3a1 1 0 0 0 0-2 .3.3 0 0 1-.3-.4v-2a1 1 0 1 0-2 0v3.4c0 .6.4 1 1 1zM36.4 73.1c.1 0 .3.2.3.4v7.4a1 1 0 1 0 2 0v-8.8c0-.6-.4-1-1-1h-1.3a1 1 0 0 0 0 2zM26.5 85c0 .6.4 1 1 1h4.8c.5 0 1-.4 1-1v-5.4a1 1 0 0 0-2 0v4c0 .3-.2.4-.4.4h-3.4a1 1 0 0 0-1 1z"/><path d="M22.7 84a1 1 0 0 0-1 1v3.4c0 .2-.1.4-.3.4H20a1 1 0 1 0 0 2h17.7c.6 0 1-.4 1-1v-4a1 1 0 0 0-2 0v2.6c0 .2-.2.4-.3.4H24a.3.3 0 0 1-.3-.4V85c0-.5-.5-1-1-1zM37.4 93.9c0-.6-.5-1-1-1h-4.1a1 1 0 0 0-1 1v5.4a1 1 0 1 0 2 0v-4c0-.3.2-.4.3-.4h2.8c.5 0 1-.5 1-1zM50 101H38.4a.3.3 0 0 1-.3-.3V98a1 1 0 1 0-2 0v4c0 .6.4 1 1 1H50a1 1 0 1 0 0-2z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M45.2 98.3h-4a1 1 0 0 1-1.1-1v-4.1c0-.6.5-1 1-1h4.1c.6 0 1 .4 1 1v4c0 .6-.4 1.1-1 1.1zm-2.7-4c-.2 0-.4 0-.4.3v1.3c0 .2.2.4.4.4h1.3c.2 0 .4-.2.4-.4v-1.3c0-.2-.2-.4-.4-.4h-1.3z"/><path d="M49.3 88a1 1 0 0 0-1 1.1V98a1 1 0 1 0 2 0V89c0-.6-.4-1-1-1zM50.3 84.3c0-.5-.4-1-1-1h-7.5a1 1 0 0 0-1 1v4.1a1 1 0 1 0 2 0v-2.7c0-.2.2-.3.4-.3h6c.7 0 1.1-.5 1.1-1z"/></g><g opacity=".3" fill="#fff"><path opacity=".7" d="M39 35.3l-1.9-3.4a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.2-.6.4l-2 3.4v.7l2 3.4c.2.2.4.4.7.4h3.9c.2 0 .5-.2.6-.4l2-3.4v-.7z"/><path opacity=".8" d="M39 54.7l-1.9-3.3a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.1-.6.4l-2 3.3v.8l2 3.3c.2.3.4.4.7.4h3.9c.2 0 .5-.1.6-.4l2-3.3v-.8z"/><path opacity=".5" d="M30.6 40.1l-1.9-3.3a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.1-.7.4l-2 3.3v.8l2 3.3c.1.3.4.4.7.4H28c.3 0 .6-.1.7-.4l2-3.3v-.8z"/><path opacity=".6" d="M30.6 49.9l-1.9-3.4a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.2-.7.4l-2 3.4v.7l2 3.4c.1.2.4.4.7.4H28c.3 0 .6-.2.7-.4l2-3.4V50z"/><path opacity=".8" d="M47.5 40.1l-2-3.3a.8.8 0 0 0-.6-.4H41c-.3 0-.5.1-.7.4L38.4 40c-.1.3-.1.6 0 .8l2 3.3c0 .3.3.4.6.4H45c.3 0 .5-.1.6-.4l2-3.3c.1-.2.1-.5 0-.8z"/><path opacity=".6" d="M47.5 49.9l-2-3.4a.8.8 0 0 0-.6-.4H41c-.3 0-.5.2-.7.4L38.4 50v.7l2 3.4c0 .2.3.4.6.4H45c.3 0 .5-.2.6-.4l2-3.4V50z"/></g><defs><radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(-67.3333 0 0 -114.941 68.7 116.3)"><stop stop-color="#FD6216"/><stop offset="1" stop-color="#FC8701"/></radialGradient><clipPath id="clip0"><path fill="#fff" transform="translate(12.9 9.6)" d="M0 0h44.1v5.3H0z"/></clipPath></defs></svg>
```
Please also take care of indentation.

File: src/request/remove-key/index.html [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <meta http-equiv="Content-Security-Policy"
-          content="default-src 'self' 'unsafe-eval' https://cdn.nimiq-testnet.com;
-          img-src http: https: blob: data:;
-          child-src 'self' blob:;
-          style-src 'self' https://fonts.googleapis.com;
-          font-src https://fonts.gstatic.com;">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-eval' https://cdn.nimiq-testnet.com;
```
Comment (nibhar) [inline src/request/remove-key/index.html]: You changed it here, but did not change it in `src/derive-address/index.html`. Why these changes?

File: src/request/remove-key/remove-key.css [hunk 6]
```diff
@@ -64,3 +57,31 @@
...
+
+.page#remove-key .backup-option > *:nth-child(2) {
+    margin-left: 3rem;
+}
+
+.page#remove-key .page-body {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+}
```
Comment (nibhar) [inline src/request/remove-key/remove-key.css]: Missing return at end of File.

File: src/request/remove-key/index.html [hunk 7]
```diff
@@ -72,28 +74,57 @@
     </div>
 
     <div id="app">
+        <!-- Remove key-->
+        <div class="page nq-card" id="remove-key">
 
-        <div class="flex-grow"></div>
```
Comment (nibhar) [inline src/request/remove-key/index.html]: removing this one, should make the last one (line 136) unnecessary, too.

File: src/request/remove-key/remove-key.css [hunk 8]
```diff
@@ -64,3 +57,31 @@
...
+
+.page#remove-key .backup-option {
+    display: flex;
+    align-items: center;
+    margin-top: 2rem;
+    max-width: 84%;
+}
+
+.page#remove-key .backup-option .login-file-svg-container {
+    min-width: 90px;
```
Comment (nibhar) [inline src/request/remove-key/remove-key.css]: Generally I would prefer `rem` over `px` in order for them to shrink with the rest of the content. These are also only `~67px` so around `8.375rem`

File: src/request/remove-key/remove-key.css [hunk 9]
```diff
@@ -64,3 +57,31 @@
 .page-footer {
     padding: 1rem;
 }
+
+.page#remove-key .backup-option {
+    display: flex;
+    align-items: center;
+    margin-top: 2rem;
```
Comment (nibhar) [inline src/request/remove-key/remove-key.css]: margin-top seems to be `5`-`5.5rem`. Additionally:
```css
.page#remove-ket .backup-option + .backup-option {
    margin-top: 3.75rem;
}
```

File: src/request/remove-key/index.html [hunk 10]
```diff
@@ -72,28 +74,57 @@
...
-                <div class="download-key-file"></div>
-                <div class="flex-grow"></div>
             </div>
 
-            <div class="page-footer hide-for-download">
-                <form class="passphrase-box"></form>
+            <div class="flex-grow"></div>
+
+            <div class="page-footer nq-card-footer">
+                <button id="remove-key-first-confirm" class="hide-for-final-confirm nq-button red"
```
Comment (nibhar) [inline src/request/remove-key/index.html]: This probably does not look like it is supposed to look?

File: src/request/remove-key/remove-key.css [hunk 11]
```diff
@@ -64,3 +57,31 @@
...
+    max-width: 84%;
```
Comment (nibhar) [inline src/request/remove-key/remove-key.css]: Where does this number come from?

File: src/translations/index.js [hunk 12]
```diff
@@ -100,27 +100,27 @@ const TRANSLATIONS = {
...
-        'remove-key-confirm': 'Log out of your wallet',
+        'remove-key-heading': 'Don\'t lose access',
+        'remove-key-intro-text': 'If you log out without saving your account, you will irretrievably '
+                                + 'lose access to it!',
+        'remove-key-login-file': 'LoginFile',
+        'remove-key-login-file-question': 'Is your LoginFile savely stored and accessible?',
+        'remove-key-download-login-file': 'Download LoginFile',
+        'remove-key-recovery-words-question': 'Do you know where your Recovery Words are?',
+        'remove-key-show-recovery-words': 'Create a backup',
+        'remove-key-first-confirm': 'I can log in again',
```
Comment (nibhar) [inline src/translations/index.js]: Missing in `de:`.
Also not the same as in DOM: `I am able to log in again`. 

I wonder why checklangs does not mention it.

File: src/request/remove-key/index.html (hunk 5 above)
Comment (Bettelstab) [reply src/request/remove-key/index.html]: No idea honestly. Will change it back

File: src/request/remove-key/index.html (hunk 10 above)
Comment (Bettelstab) [reply src/request/remove-key/index.html]: Right. Soeren and me weren't quite happy with the 'checkbox-button' style. There will be another solution probably.

File: src/request/remove-key/remove-key.css (hunk 11 above)
Comment (Bettelstab) [reply src/request/remove-key/remove-key.css]: Experiments.

File: src/request/remove-key/index.html [hunk 13]
```diff
@@ -73,28 +76,55 @@
...
...-1.3z"/><path d="M49.3 88a1 1 0 0 0-1 1.1V98a1 1 0 1 0 2 0V89c0-.6-.4-1-1-1zM50.3 84.3c0-.5-.4-1-1-1h-7.5a1 1 0 0 0-1 1v4.1a1 1 0 1 0 2 0v-2.7c0-.2.2-.3.4-.3h6c.7 0 1.1-.5 1.1-1z"/></g><g opacity=".3" fill="#1F2348"><path opacity=".7" d="M39 35.3l-1.9-3.4a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.2-.6.4l-2 3.4v.7l2 3.4c.2.2.4.4.7.4h3.9c.2 0 .5-.2.6-.4l2-3.4v-.7z"/><path opacity=".8" d="M39 54.7l-1.9-3.3a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.1-.6.4l-2 3.3v.8l2 3.3c.2.3.4.4.7.4h3.9c.2 0 .5-.1.6-.4l2-3.3v-.8z"/><path opacity=".5" d="M30.6 40.1l-1.9-3.3a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.1-.7.4l-2 3.3v.8l2 3.3c.1.3.4.4.7.4H28c.3 0 .6-.1.7-.4l2-3.3v-.8z"/><path opacity=".6" d="M30.6 49.9l-1.9-3.4a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.2-.7.4l-2 3.4v.7l2 3.4c.1.2.4.4.7.4H28c.3 0 .6-.2.7-.4l2-3.4V50z"/><path opacity=".8" d="M47.5 40.1l-2-3.3a.8.8 0 0 0-.6-.4H41c-.3 0-.5.1-.7.4L38.4 40c-.1.3-.1.6 0 .8l2 3.3c0 .3.3.4.6.4H45c.3 0 .5-.1.6-.4l2-3.3c.1-.2.1-.5 0-.8z"/><path opacity=".6" d="M47.5 49.9l-2-3.4a.8.8 0 0 0-.6-.4H41c-.3 0-.5.2-.7.4L38.4 50v.7l2 3.4c0 .2.3.4.6.4H45c.3 0 .5-.2.6-.4l2-3.4V50z"/></g><defs><clipPath id="clip0"><path fill="#fff" transform="translate(12.9 9.6)" d="M0 0h44.1v5.3H0z"/></clipPath></defs></svg>
```
Comment (nibhar) [inline src/request/remove-key/index.html]: I would prefer if you would apply proper indentations to elements and add some line breaks to get rid of a 3404 character line.

**Comment by Bettelstab** [comment]
Note: Julian said he would work out a design for entering the account's name later, so I'll wait for that to integrate it here.

File: src/request/remove-key/index.html [hunk 14]
```diff
@@ -73,28 +76,65 @@
...
```
Comment (sisou) [inline src/request/remove-key/index.html]: You now moved the svg file to extern, but you left the inline-svg?

File: src/request/derive-address/index.html [hunk 15]
```diff
@@ -67,13 +67,13 @@
```
Comment (sisou) [inline src/request/derive-address/index.html]: Please remove all changes of this file, as this is done in #191 and will lead to merge conflicts.

File: src/request/remove-key/index.html [hunk 16]
```diff
@@ -48,7 +49,8 @@
     <script defer src="index.js"></script>
 
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Muli:400,600,700">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY">
```
Comment (sisou) [inline src/request/remove-key/index.html]: Why add a line break? This is done in no other index.html.

File: src/request/remove-key/index.html [hunk 17]
```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html>
+
```
Comment (sisou) [inline src/request/remove-key/index.html]: Why add an empty new line?

File: src/request/remove-key/index.html [hunk 18]
```diff
@@ -73,28 +76,56 @@
...
                 </div>
-                <div class="download-key-file"></div>
-                <div class="flex-grow"></div>
             </div>
 
-            <div class="page-footer hide-for-download">
-                <form class="passphrase-box"></form>
+            <div class="page-footer nq-card-footer">
+                <div id="remove-key-label-confirm" class="hide-for-final-confirm">
+                    <div>Type "<span></span>" to log out.</div>
```
Comment (sisou) [inline src/request/remove-key/index.html]: Missing `data-i18n` tags for translation here.

File: src/request/remove-key/remove-key.css [hunk 19]
```diff
@@ -37,30 +37,62 @@
...
 }
 
-.page-footer {
-    padding: 1rem;
+.page#remove-key .page-footer button {
+    margin-top: 0;
+    width: calc(100% - 7rem);
+}
+
+.page#remove-key.page#remove-key:not(.show-final-confirm) .page-footer #remove-key-label-confirm {
```
Comment (sisou) [inline src/request/remove-key/remove-key.css]: Double `.page#remove-key` here. And generally very high specificity here. Is that necessary?

File: src/request/remove-key/remove-key.css [hunk 20]
```diff
@@ -37,30 +37,62 @@
...
+    width: 100%;
+    height: 11.25rem;
+    background-color: var(--nimiq-red);
+    color: #fff;
```
Comment (sisou) [inline src/request/remove-key/remove-key.css]: When you use the `.nq-red-bg` class on the element, you don't need these two rules I think.

File: src/request/remove-key/remove-key.css [hunk 21]
```diff
@@ -37,30 +37,62 @@
...
+    font-style: normal;
+    font-weight: 600;
+    line-height: 21px;
+    font-size: 14px;
```
Comment (sisou) [inline src/request/remove-key/remove-key.css]: Use `rem`.

File: src/request/remove-key/remove-key.css [hunk 22]
```diff
@@ -37,30 +37,62 @@
...
+    margin-bottom: 0;
+    border-radius: 4px;
```
Comment (sisou) [inline src/request/remove-key/remove-key.css]: Use `rem`.

File: src/translations/index.js [hunk 23]
```diff
@@ -102,27 +102,27 @@ const TRANSLATIONS = {
...
-                               + 'Recovery Words accessible.',
-        'remove-key-intro-text-red': 'If you have neither of them, there’s no chance to regain '
-                                   + 'access to your wallet.',
-        'remove-key-key-file': 'Wallet File',
-        'remove-key-download-key-file': 'Download Wallet File',
-        'remove-key-show-recovery-words': 'Show Recovery Words',
-        'remove-key-confirm': 'Log out of your wallet',
+        'remove-key-heading': 'Don\'t lose access',
+        'remove-key-intro-text': 'If you log out without saving your account, you will irretrievably '
+                                + 'lose access to it!',
```
Comment (sisou) [inline src/translations/index.js]: One too many space before `+`.

File: src/translations/index.js [hunk 24]
```diff
@@ -102,27 +102,27 @@ const TRANSLATIONS = {
...
+        'derive-address-passphrase-text': 'Please enter your password to add another address to your account.',
+        'derive-address-heading-choose-identicon': 'Choose your address avatar',
+        'derive-address-text-select-avatar': 'Select an avatar for your new address from the selection below.',
 
-        'change-passphrase-heading': 'Change your passphrase',
-        'change-paragraph': 'Before entering your new passphrase, you must first unlock your wallet.',
-        'change-set-paragraph': 'Now please enter your new Passphrase and repeat it for confirmation.',
+        'change-passphrase-heading': 'Change your password',
+        'change-paragraph': 'Before entering your new password, you must first unlock your account.',
+        'change-set-paragraph': 'Now please enter your new password and repeat it for confirmation.',
```
Comment (sisou) [inline src/translations/index.js]: You did not change the wording in the german translation.

File: src/request/remove-key/index.html (hunk 17 above)
Comment (Bettelstab) [reply src/request/remove-key/index.html]: (*I thought you liked them*)
My editor adds it when autoformating. I guess it's kind of a replacement for the missing indent, and adds some visual structure. But I don't care too much.

**Comment by Bettelstab** [comment]
Done.
Missing #197, but I'd like to do that one later as I think there are things with higher priority.

File: src/request/remove-key/RemoveKeyApi.js [hunk 25]
```diff
@@ -33,7 +33,13 @@ class RemoveKeyApi extends TopLevelApi { // eslint-disable-line no-unused-vars
         const parsedRequest = {};
         parsedRequest.appName = this.parseAppName(request.appName);
         parsedRequest.keyInfo = await this.parseKeyId(request.keyId);
-        parsedRequest.keyLabel = this.parseLabel(request.keyLabel);
+        const parsedLabel = this.parseLabel(request.keyLabel);
+
+        if (parsedLabel === undefined) {
+            throw new Errors.InvalidRequestError('Label must be a string');
```
Comment (sisou) [inline src/request/remove-key/RemoveKeyApi.js]:
```suggestion
            throw new Errors.InvalidRequestError('keyLabel must be a string');
```

File: src/request/remove-key/index.html (hunk 1 above)
Comment (sisou) [reply src/request/remove-key/index.html]: This still.

File: src/translations/index.js [hunk 26]
```diff
@@ -102,27 +102,29 @@ const TRANSLATIONS = {
...
         'remove-key-back': 'Back to logout',
 
-        'derive-address-heading-passphrase': 'Unlock your wallet',
-        'derive-address-passphrase-text': 'Please enter your passphrase to add another account to your wallet.',
-        'derive-address-heading-choose-identicon': 'Choose your account avatar',
-        'derive-address-text-select-avatar': 'Select an avatar for your new account from the selection below.',
+        'derive-address-heading-passphrase': 'Unlock your account',
+        'derive-address-passphrase-text': 'Please enter your password to add another address to your account.',
+        'derive-address-heading-choose-identicon': 'Choose your address avatar',
+        'derive-address-text-select-avatar': 'Select an avatar for your new address from the selection below.',
```
Comment (sisou) [inline src/translations/index.js]: There's still `derive-address` related changes here.
